model/clusters_mgmt/v1/azure_type.model

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Microsoft Azure settings of a cluster.
+struct Azure {
+	// [Required] The Azure Subscription ID associated with the cluster. It must belong to `tenant_id`.
```

Comment:
Unfortunately we can't add a `required` OpenAPI attribute with the metamodel so we document it